gradle-enterprise-conventions-maven-extension/src/main/java/com/example/CiUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.example;
+
+import java.util.Optional;
+
+final class CiUtils {
+
+    private CiUtils() {
+    }
+
+    static boolean isCi() {
```

Comment:
We already implement this (and more thoroughly) in the Common Custom User Data Maven Extension: https://github.com/gradle/common-custom-user-data-maven-extension/blob/main/src/main/java/com/gradle/CiUtils.java#L8

If we exposed that, we could use it here and ditch this whole class